vendor/opam/src/core/dune

```diff
@@ -0,0 +1,45 @@
+(library
+  (name        opam_core)
+  (synopsis    "OCaml Package Manager core internal stdlib")
+  (libraries   dune_re unix bigarray sha uutf
+               (select opamACL.ml from
```

Comment:
What's this acl stuff? If we don't need it, then let's remove it because we don't support `select` in bootstrapping.